bin/cli.js

```diff
@@ -1,9 +1,171 @@
 #!/usr/bin/env node
+'use strict';
```

Comment:
ESM mode implies strict mode? I don't think that this is needed

```suggestion
```

Reply:
Done, thanks for spotting 😄